src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.jsx

```diff
@@ -1,7 +1,137 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 
-const ClaimsAndAppeals = () => {
-  return <h2>Claims & appeals</h2>;
+import backendServices from '~/platform/user/profile/constants/backendServices';
+import { createIsServiceAvailableSelector } from '~/platform/user/selectors';
+
+import {
+  getAppealsV2 as getAppealsAction,
+  getClaimsV2 as getClaimsAction,
+} from '~/applications/claims-status/actions';
+
+import useOpenClaimsAppealsCount from './hooks/useOpenClaimOrAppealCount';
+import useHighlightedClaimOrAppeal from './hooks/useHighlightedClaimOrAppeal';
+
+const ClaimsAndAppealsCTA = ({ count }) => {
+  let content = 'Go to all claims or appeals';
+  if (count === 1) {
+    content = '1 claim or appeal in progress';
+  } else if (count > 1) {
+    content = `${count} claims or appeals in progress`;
+  }
+  return <a href="claim-or-appeal-status/">{content}</a>;
+};
+
+const HighlightedClaimAppeal = ({ claimOrAppeal }) => {
+  if (!claimOrAppeal) {
+    return <p>You have no claims or appeals updates in the last 30 days.</p>;
+  } else {
+    return (
+      <div>
+        <p>Show some details about the claim or appeal</p>
+        <a className="usa-button-primary" href="claim-or-appeal-status/">
+          View details
+        </a>
+      </div>
+    );
+  }
+};
+
+const ClaimsAndAppeals = ({
+  appealsData,
+  claimsData,
+  // for some unit testing purposes, we want to prevent this component from
+  // making off API calls which kicks off a chain of events that results in the
+  // component always showing a loading spinner
+  dataLoadingDisabled = false,
+  loadAppeals,
+  loadClaims,
+  shouldLoadAppeals,
+  shouldLoadClaims,
+  shouldShowLoadingIndicator,
+}) => {
+  React.useEffect(
+    () => {
+      if (!dataLoadingDisabled && shouldLoadAppeals) {
+        loadAppeals();
+      }
+    },
+    [dataLoadingDisabled, loadAppeals, shouldLoadAppeals],
+  );
+
+  React.useEffect(
+    () => {
+      if (!dataLoadingDisabled && shouldLoadClaims) {
+        loadClaims();
+      }
+    },
+    [dataLoadingDisabled, loadClaims, shouldLoadClaims],
+  );
+
+  // the most recently updated claim or appeal that has been updated in the past
```

Comment:
I like your comments. Makes me reevaluate that I also might need some clarifications within my appointments PR.